apps/wallet/src/ui/app/index.tsx

```diff
@@ -57,10 +67,28 @@ const App = () => {
 	}, [location, dispatch]);
 
 	useInitialPageView();
+	useEffect(() => {
+		(async () => {
+			const interstitialDismissed = await get<boolean>('bullshark-interstitial-dismissed');
+			if (interstitialDismissed) {
+				setInternalInterstitialDismissed(interstitialDismissed);
+			}
+		})();
+	}, []);
 
 	return (
 		<Routes>
 			<Route path="/*" element={<HomePage />}>
+				<Route
```

Comment:
why is this a route? Seems like it really should just be an overlay on the tokens page?